tests/unit/test_auth_model.py

```
@@ -0,0 +1,93 @@
+    )
+    data = vo.dict()
+    assert data["account_type"] == "GOOGLE"
+    assert isinstance(data["account_type"], str)
+    assert data["oauth_id"] == "google-oauth-id-123"
+
+
+def test_new_account_dto_gen_account_entity_xc_has_required_fields():
+    dto = NewAccountDTO(
+        region="TW",
+        email="test@example.com",
+        password="Password123!",
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "password"

(S106)